packages/open-banking/mx/README.md

```diff
@@ -0,0 +1,52 @@
+# Dwolla and MX - Open Banking
+
+This example project, built using [Next.js](https://nextjs.org), demonstrates open banking integration between Dwolla and MX. The app showcases how to create a Customer in Dwolla and attach a verified Funding Source using MX's integration with Dwolla, enabling secure exchange of financial information without handling sensitive data directly.
+
+**Note**: Since this project depends on shared dependencies, please ensure that you have executed `pnpm install` in the root directory (`integration-examples`) before proceeding.
+
+## Prerequisites
+
+1. Create a [Dwolla Sandbox Account](https://accounts-sandbox.dwolla.com/sign-up) and obtain the necessary API keys.
+
+## Setup
+
+1. Rename `.env.local.example` to `.env.local`, and enter the necessary access keys and environment settings for Dwolla and MX.
+2. Run `pnpm install` to download all necessary dependencies.
+3. Run `pnpm dev` to start the Next.js application! (Before connecting a bank account, please see _[Using MX Test Credentials](#using-mx-test-credentials)_.)
+
+## Using MX Connect
+
+When using MX Web Widget SDK to connect a bank account, there are two development options present that work with Dwolla: **MX Bank** and **MX Bank (OAuth)**. 
+
+Please keep in mind, to enable **MX Bank (OAuth)**, you must register through the MX Dashboard by completing the provided form. Please see [MX's website](https://docs.mx.com/resources/test-platform/) for all the different testing scenarios and credentials.
```

Comment:
@ShreyaThapa Does this apply since the integration is managed by Dwolla and not the client? Assuming they won't be able to configure this

Reply:
Ah! Good call out. Yeah, this wouldn't apply to Open Banking with MX.

Thanks for catching this!  Addressed here - [c599069](https://github.com/Dwolla/integration-examples/pull/36/commits/c5990698811ba8c064872ab18482f09b01a6ec0d)